shared/instance.go

```diff
@@ -1206,6 +1206,15 @@ func ConfigKeyChecker(key string, instanceType instancetype.Type) (func(value st
 			return validate.IsAny, nil
 		}
 
+		// lxdmeta:generate(entity=instance, group=volatile, key=volatile.<name>.<target_path>.dir_created_by_lxd)
+		//
+		// ---
+		//  type: bool
+		//  shortdesc: if the disk device mounted from a host directory, created a new directory within the instance
```

Comment:
Please split that up into a short description and a longer description. If you look at the output, the short description takes two lines (which is not "short" ;) ).
Also, I don't really understand what the option does from this description. Try to go for just the essence (maybe "How the host directory was created" or something?) for the short description, and then clearly describe what it means in the longer description.